src/rard/research/forms.py

```diff
@@ -67,6 +67,9 @@ def __init__(self, *args, **kwargs):
             self.fields[
                 "introduction_text"
             ].initial = self.instance.introduction.content
+            self.fields["introduction_text"].widget.attrs[
+                "class"
+            ] = "enableMentions enableCKEditor"
```

Comment:
At the moment enableFootnotes is only applied to the Commentaries, should they be enabled on Introductions (Antiquarian, Work, Book...) as well? I can't remember.

Reply:
seems issue #392 says it should be, I'll update that